Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -46,24 +46,19 @@
   final double TAGS_SCORE_WEIGHT = 0.75;
   final double AVG_REVIEW_WEIGHT = 0.25;
 
-  private static Collection<Charity> charities;
+  // collection that holds charities from the db
```

Comment:
this file is related to trending, so why are we touching this?

Reply:
Aneri and I agreed on removing the if statement that checks whether to re-populate the db from FindTrendingCharities. The other changes I made were just a few minor formatting changes.

Reply:
In the deployed version, we will always populate the db at first using the script call so the if statement becomes redundant. My branch may make some other changes to the constructor logic as well, just to make it run more efficiently.